src/mocks/mock-gmail.js

```
@@ -0,0 +1,153 @@
+};
+
+/**
+ * Get attachment for a specific email
+ * @param {Object} params - Parameters
+ * @param {string|number} params.emailId - Email ID or index
+ * @param {string} params.attachmentName - Name of attachment
+ * @param {string} [params.samplePdfsDir] - Custom path to sample PDFs
+ * @returns {Promise<Object>} Result with attachment data
+ */
+const getAttachment = ({ 
+  emailId, 
```

ESLint

[error] 'emailId' is defined but never used.

(no-unused-vars)